test/lib/godmin/model/scopes_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+module Godmin
+  module Model
+    class ScopesTest < ActiveSupport::TestCase
+      def setup
+        @article_thing = ArticleThing.new
+      end
+
+      def test_calls_default_scope
+        @article_thing.apply_scope("", :resources)
+        assert_equal :resources, @article_thing.called_methods[:scopes][:unpublished]
+      end
+
+      def test_calls_non_default_scope
+        @article_thing.apply_scope("published", :resources)
+        assert_equal :resources, @article_thing.called_methods[:scopes][:published]
+      end
+
+      def test_calls_unimplemented_scope
+        assert_raises NotImplementedError do
+          @article_thing.apply_scope("trashed", :resources)
+        end
+      end
+
+      def test_current_scope
+        @article_thing.apply_scope("", :resources)
+        assert_equal "unpublished", @article_thing.scope
+      end
+
+      def test_currently_scoped_by
+        @article_thing.apply_scope("", :resources)
+        assert     @article_thing.scoped_by?("unpublished")
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.